EventsExpress/ClientApp/src/reducers/user.js

```diff
@@ -13,12 +13,12 @@ export const reducer = (state = initialState.user, action) => {
         case SET_LOGOUT:
             return initialState.user;
 
-        case addUserCategory.UPDATE && GET_USER_CATEGORIES_DATA:
+        case addUserCategory.UPDATE || GET_USER_CATEGORIES_DATA:
```

Comment:
I guess smth like:
```
case addUserCategory.UPDATE:
case GET_USER_CATEGORIES_DATA:
   return { ...
```
or in jsthis works in some other way?

Reply:
You are correct - this needs to be fixed, b/c the sonar also says it's wrong.